modules/api/src/main/java/org/eclipse/xpanse/api/config/CaffeineCacheConfig.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import java.util.concurrent.TimeUnit;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * The configuration class of Caffeine Cache.
+ */
+@Configuration
+@EnableCaching
+public class CaffeineCacheConfig {
+
+    @Value("${caffeine.cache.expire.time}")
+    private long duration;
+
+    /**
+     * Create the configured CaffeineCacheManager.
+     *
+     * @return CaffeineCacheManager
+     */
+    @Bean
+    public CaffeineCacheManager caffeineCacheManager() {
+        CaffeineCacheManager cacheManager = new CaffeineCacheManager("caffeine_cache");
```

Comment:
It's better to replace `caffeine_cache` with a meaningful name, like `region_azs_cache`.